migrations/migrations/20210601230017-users-points-column.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('users', 'points', {
+      type: Sequelize.INTEGER,
+      defaultValue: 0,
+      allowNull: true
```

Comment:
If there is a default value, I think it would be better to be notNull. What do you think?